test/helpers/ReserveOracleSharedSetup.sol

```diff
@@ -37,19 +38,29 @@ contract ReserveOracleSharedSetup is IonPoolSharedSetup {
     bytes32 constant STADER_ORACLE_TOTAL_SUPPLY_SLOT =
         0x0000000000000000000000000000000000000000000000000000000000000104;
 
+    bytes32 constant EETH_LIQUIDITY_POOL_TOTAL_VALUE_SLOT =
+        0x00000000000000000000000000000000000000000000000000000000000000cf; // uint128 totalValueInLp, uint128
+        // totalValueOutOfLp
+    bytes32 constant EETH_TOTAL_VALUE_MASK = 0x00000000000000000000000000000000FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF;
+
     // fork configs
 
     string public MAINNET_RPC_URL = vm.envString("MAINNET_ARCHIVE_RPC_URL");
 
     uint256 constant BLOCK_NUMBER = 18_372_927;
 
+    uint256 public blockNumber;
+
+    function setBlockNumber(uint256 _blockNumber) public {
+        blockNumber = _blockNumber;
+    }
+
```

Comment:
Since `blockNumber` is public variable, why not skip the setter and just set it in the child?